src/LitVocabTerm.ts

```diff
@@ -185,6 +185,12 @@ class LitVocabTerm implements NamedNode {
     return this.value;
   }
 
+  // Get the IRI of this term as a String (means we can treat this object
+  // instance as a string more easily).
+  get toString(): string {
```

Comment:
```suggestion
  toString(): string {
```
This does not really resolve the issue we wanted to address in my opinion: with `toString` being a get accessor, and not a method, `'${myTerm}'` doesn't return the string IRI, and in plain JS `myMap[myTerm]` fails. My thinking was to override the method that gets called on the `Object` type in these cases, which is `toString()`, and not `toString`.

Reply:
Ah, gotcha! I didn't know that distinction - I assumed the Typescript accessor was just syntactic sugar over the method override. Thanks!